postcss.config.js

```diff
@@ -15,7 +15,10 @@ module.exports = ({ file }) => {
   if (isProd && shouldPurgeFile) {
     // see https://purgecss.com/guides/vue.html for reference
     config.plugins['@fullhuman/postcss-purgecss'] = {
-      content: ['./src/**/*.vue', './src/**/*.story.js'],
+      content: [
+        './src/**/*.vue',
+        './src/**/*.story.js',
+        './.storybook/*.js'],
```

Comment:
@jaimeperezpedra This will change to './.preview/*.js' after you merge your pr

Reply:
the postcss.config.js is not changing, I changed the config.js from the .storybook folder